bugbug/model.py

```diff
@@ -402,12 +407,15 @@ def train(self, importance_cutoff=0.15):
         # Evaluate results on the test set for some confidence thresholds.
         for confidence_threshold in [0.6, 0.7, 0.8, 0.9]:
             y_pred_probas = self.clf.predict_proba(X_test)
+            confidence_class_names = self.class_names + ["NOT_CLASSIFIED"]
 
             y_test_filter = []
             y_pred_filter = []
+            not_classified_indices = []
```

Comment:
This implementation seems more complex than necessary, couldn't you simply `y_pred_filter.append("___NOT_CLASSIFIED___")` when `y_pred_probas[i][argmax] < confidence_threshold`?

Reply:
This is the problem https://github.com/mozilla/bugbug/blob/6426361dbff7e06a4f677af4b631687005e1fc9f/bugbug/model.py#L420

Reply:
OK, makes sense.